### DISCUSSION TITLE
Allow labelsType to be "name" and not only "value" or "percent"

### DISCUSSION THREAD

**Discussion by dnng** [mantinedev/mantine#6626] [Feature requests]
It would be nice if I could easily add the name of each section to a Pie Chart.

**Comment by josch0** [comment]
In my opinion, “name” should be the default label of a pie chart. I want to use a pie chart to show how the individual items are distributed across the whole. Therefore, the absolute “value” does not matter to me. I can recognize the approximate “percentage” based on the size of the pie slice. But I lack the name at first glance so that I can identify the pie pieces.
